example/relational/README.md

```diff
@@ -78,7 +78,7 @@ dynamo.NewIRI("keyword:haskell")
 The data type `dynamo.IRI` (`curie.IRI`) makes formal rules of building keys from multiple segments. The application has a common interface to construct keys of any complexity to resolve data sharding aspects. 
 
 ```go
-dynamo.NewIRI("author:smith/%d", 1)
+dynamo.NewIRI("author:smith/%d", 1) => wouldn't it be better using random uuid?
```

Comment:
This is just an example... may be it has to be defined as `Smith Joe` and `Smith John`. However, there are not requirements to use "random" primary key. DynamoDB internally calculates hash from primary key value to make decision about allocation to the partition.  Highly likely `author:smith/1` and `author:smith/2` ends up to different paritions.